services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -414,6 +416,8 @@ def update_stac_item_publication(  # pylint: disable=too-many-locals
                 old_bucket_arr[2] = CATALOG_BUCKET
                 s3_key = "/".join(old_bucket_arr)
                 # Check if the S3 key exists
+                # TBD: The catalog should have nothing to do anymore with the
```

Comment:
Then remove it and update tests

Reply:
This should happen after the adgs is aligned with the new staging process. A story should be created for this